components/Withdraw.tsx

```diff
@@ -133,7 +133,7 @@ const Withdraw: React.FC<{}> = () => {
           await api.rpc.ethy.getEventProof(eventProofId)
         ).toJSON();
         if (versionedEventProof !== null) {
-          eventProof = versionedEventProof.EventProof;
+          eventProof = versionedEventProof.eventProof;
```

Comment:
Are there any more of this chaging case is happening across the site @aidan-starke ?

Reply:
this is the only one I have found, with this change everything is working well

Reply:
I think that PR using cennznet/api 2.x.x version. in 1.5.x still lowercase. caused by polkadot/api changes in between the versions

Reply:
I think that PR using cennznet/api 2.x.x version. in 1.5.x still lowercase. caused by polkadot/api changes in between the versions

Reply:
@aidan-starke we have a derived query which will give you the proof once it is ready... 
https://github.com/cennznet/api.js/blob/master/packages/api/src/derives/ethBridge/eventProof.ts#L27
Usage - https://github.com/cennznet/bridge-contracts/blob/main/scripts/subscribeEventProof.js#L50

Reply:
this will also update the signature in r,s,v format... right now you would just need to add the current validators list to it.. But I have it in my to do list to add the validators list to the event proof from derived query so that you can just use it as it